stdlib/bytes.mli

```diff
@@ -463,6 +463,106 @@ val of_seq : char Seq.t -> t
 (** Create a string from the generator
     @since 4.07 *)
 
+(** {6 Binary storage of integers} *)
+
+val get_uint16_ne : bytes -> int -> int
+(** [get_uint16_ne s n] reads a 16-bit unsigned integer from the byte
+    sequence [s] starting at index [n], in binary form, using the
+    machine native endianness. The result is an integer between 0 and
```

Comment:
I don't think it's worth spelling out the limits of `int16` and/or `uint16` it should be pretty clear to a programmer what to expect I think.